plugin.json

```diff
@@ -23,12 +23,9 @@
       "header": "Configure this plugin directly in the config.json file. Learn more [in our documentation](https://github.com/mattermost/mattermost-plugin-welcomebot/blob/master/README.md).\n\n To report an issue, make a suggestion, or submit a contribution, [check the plugin repository](https://github.com/mattermost/mattermost-plugin-welcomebot).",
       "settings": [
         {
-            "key": "ExistingConfigurationTable",
-            "display_name": "Existing Configurations:",
+            "key": "WelcomeMessages",
```

Comment:
it should be in `camelCase` 
this is suggested by MM as well https://developers.mattermost.com/integrate/plugins/manifest-reference/

Reply:
key has been updated to 'WelcomeMessages'. For this plugin, the struct in server is made with the name of the key, and the server side of the plugin is based on struct named as 'WelcomeMessages' with a capital W. if we change the key to camel case, we will need to update the server also. should i proceed ?

Reply:
In that case we can keep it

Reply:
sure